cmd/sampledata/main.go

```diff
@@ -115,7 +117,7 @@ func populateScans(store interfaces.IStorage) error {
 			nil,
 			scan.ID,
 			enums.StatusCompleted.String(),
-			scan.StartedAt.Add(time.Minute*5),
+			scan.StartedAt.Add(time.Duration(gofakeit.IntRange(1, 100))),
```

Comment:
It does seem to generate some weird results though, please consider:

![image](https://github.com/user-attachments/assets/b1db00bf-9190-4ee3-ab9d-11f1e6e896fa)




```suggestion
			scan.StartedAt.Add(time.Minute * gofakeit.IntRange(1, 100)),
```